.github/workflows/create_additional_release_tags.yaml

```diff
@@ -27,10 +27,17 @@ jobs:
 
     - name: Create additional tags
       run: |
+        REF=${{ github.event.release.tag_name || github.event.inputs.releaseTag }}
         ARTIFACT_IDS=('google-cloud-shared-dependencies' 'api-common' 'gax')
         for ARTIFACT_ID in "${ARTIFACT_IDS[@]}"
         do
-          VERSION=$(grep "${ARTIFACT_ID}" versions.txt | cut -d':' -f2)
-          git tag ${ARTIFACT_ID}/$VERSION ${{ github.event.inputs.releaseTag }}
-          git push origin ${ARTIFACT_ID}/$VERSION
+          VERSION=$(grep "^${ARTIFACT_ID}:" versions.txt | cut -d':' -f2 | tr -d '[:space:]')
+          TEMP_BRANCH="temp_branch_${ARTIFACT_ID}_$VERSION"
```

Comment:
I don't think you need a branch to create tags.

Reply:
Can you show me the permission error that was resolved by creating a branch or deleting the yaml file?